.gitlab-ci.yml

```diff
@@ -37,3 +38,13 @@ end:
   only:
     refs:
       - external_pull_requests
+
+notify_failure:
+  stage: notify
```

Comment:
I don't think we need a new stage. "end" stage is the end. Nothing else happens there (it's actually empty job). So we can put this there